src/components/SkillRatingCard/SkillRatingCard.js

```diff
@@ -13,19 +17,32 @@ import {
   Legend,
   ResponsiveContainer,
 } from 'recharts';
-import { Paper } from 'material-ui';
 import Ratings from 'react-ratings-declarative';
-import Cookies from 'universal-cookie';
-import PropTypes from 'prop-types';
+import { Paper } from 'material-ui';
```

Comment:
Make a separate section for Material-UI, as in some components there are multiple such imports.